pkg/cluster/install.go

```diff
@@ -150,6 +151,14 @@ func (m *manager) adminUpdate() []steps.Step {
 	return toRun
 }
 
+func (m *manager) clusterWasCreatedByHive() bool {
```

Comment:
Would something simpler work, or are there cases where `Namespace` is not empty but`CreatedByHive` is false?

```suggestion
func (m *manager) clusterWasCreatedByHive() bool {
	return m.doc.OpenShiftCluster.Properties.HiveProfile.Namespace != "" 
}
```

Reply:
Good question. 
Short answer: no, we can't.
Long answer:

As this discussed in the slack [thread](https://redhat-internal.slack.com/archives/CE3ETN3J8/p1673621733329919), checking just for the namespace is not enough. We need to differentiate if it was created by hive vs adopted by hive.

As hive doesn't have this information, wee need to keep track of that. That's why we (@UlrichSchlueter and me) decided to place the information in the cluster document in CosmosDB and in a label in the cluster deployment during install time (when the cluster is created via Hive).